packages/cmf/src/sagas/component.js

```diff
@@ -24,11 +23,9 @@ export function get(id, context) {
 
 export const registerMany = registry.getRegisterMany(register);
 
-export const isActionCancelable = curry(
-	(startAction, action) =>
-		action.type === `${CONST.WILL_UNMOUNT_SAGA_STOP}_${startAction.saga}` &&
-		startAction.event.componentId === action.event.componentId,
-);
+export const isActionCancelable = startAction => action =>
```

Comment:
just remove the curry. It seems that there is a issue with the TAKE to check the random componentId and not the componentId from the settings.

replace the take call by yield take(actionTAKE => isActionCancelable(action, actionTAKE));